components/Layout/AudioControls.vue

```diff
@@ -1,135 +1,315 @@
+            </v-tooltip>
+            <item-menu :item="getCurrentItem" :absolute="false" :dark="false" />
+          </v-col>
+          <v-col
+            cols="3"
+            class="d-flex d-md-none px-0 align-center justify-end"
           >
-            {{ getCurrentItem.AlbumArtist }}
-          </nuxt-link>
-        </div>
-      </v-col>
-      <v-col cols="6" class="pa-0 d-none d-md-inline">
-        <div class="d-flex flex-column justify-center">
-          <div class="d-flex align-center justify-center">
-            <v-btn disabled icon class="mx-1">
-              <v-icon>mdi-shuffle</v-icon>
-            </v-btn>
-            <v-btn icon class="mx-1" @click="setPreviousTrack">
-              <v-icon>mdi-skip-previous</v-icon>
-            </v-btn>
-            <v-btn icon large class="mx-1" @click="togglePause">
-              <v-icon large>
+            <v-btn
+              elevation="6"
+              fab
+              icon
+              raised
+              rounded
+              class="mx-1"
+              :class="isPaused ? '' : 'outlined-button'"
+              @click="togglePause"
+            >
+              <v-icon>
                 {{ isPaused ? 'mdi-play' : 'mdi-pause' }}
               </v-icon>
             </v-btn>
-            <v-btn icon class="mx-1" @click="stopPlayback">
-              <v-icon>mdi-stop</v-icon>
-            </v-btn>
-            <v-btn icon class="mx-1" @click="setNextTrack">
+            <v-btn
+              icon
+              :disabled="!getNextItem"
+              class="mx-1"
+              @click="setNextTrack"
+            >
               <v-icon>mdi-skip-next</v-icon>
             </v-btn>
-            <v-btn disabled icon class="mx-1">
-              <v-icon>mdi-repeat-off</v-icon>
+            <v-btn
+              icon
+              fab
+              small
+              :elevation="isRepeating ? '3' : '0'"
+              class="mx-1"
+              :class="isRepeating ? '' : 'text--disabled'"
+              :color="isRepeating ? 'primary' : undefined"
+              @click="toggleRepeatMode"
+            >
+              <v-icon>{{ repeatIcon }}</v-icon>
+            </v-btn>
+            <v-btn
+              icon
+              fab
+              small
+              :elevation="isShuffling ? '3' : '0'"
+              class="mx-1"
+              :class="isShuffling ? '' : 'text--disabled'"
+              :color="isShuffling ? 'primary' : 'text--disabled'"
+              @click="toggleShuffle"
+            >
+              <v-icon>mdi-shuffle</v-icon>
             </v-btn>
+            <item-menu :item="getCurrentItem" :absolute="false" :dark="false" />
+          </v-col>
+        </v-row>
+        <div
+          v-if="isFullScreenPlayer && getNextItem"
+          class="d-flex justify-center align-center"
+        >
+          <div>
+            <h4 class="text-overline font-italic">
+              {{ $t('upNext') }}: {{ getNextItem.Name }}
```

Comment:
Make this whole line translated like we have done in Card.vue L233

`$t('upNext', {upNextItemName: getNextItem.Name})`